authors/apps/article/tests/test_favorite_article.py

```diff
@@ -0,0 +1,158 @@
+                                            format='json')
+        article = json.loads(response_article.content)
+
+        self.assertIn(
+            'Successfully favourited the article', str(article))
+        self.assertEqual(response_article.status_code, status.HTTP_202_ACCEPTED)
+
+    def test_double_favorite(self):
+        """Tests a request for succesfull favoriting an article twice
+            :return:
+        """
+        response = self.client.post('/api/articles/', self.article,
+                                    HTTP_AUTHORIZATION='Token ' + self.token,
+                                    format='json')
+        result = json.loads(response.content)
+        slug = result["article"]["slug"]
+
+        self.client.post('/api/articles/' + slug + '/favorite',
+                        HTTP_AUTHORIZATION='Token ' + self.token,
+                        format='json')
+
+        response_article = self.client.post('/api/articles/' + slug + '/favorite',
+                                            HTTP_AUTHORIZATION='Token ' + self.token,
+                                            format='json')
+        article = json.loads(response_article.content)
+
+        self.assertIn(
+            'you have already favourited the article', str(article))
+        self.assertEqual(response_article.status_code, status.HTTP_406_NOT_ACCEPTABLE)
+
+    def test_favorite_unexisting_article(self):
+        """
+        test if slug is created in every instance
+        :return:
+        """
+        response_article = self.client.post('/api/articles/test/favorite',
+                                            HTTP_AUTHORIZATION='Token ' + self.token,
+                                            format='json')
+        article = json.loads(response_article.content)
+
+        self.assertIn(
+            "You can not favourite this article because it doesn't exist", str(article))
+        self.assertEqual(response_article.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_delete_favorite(self):
+        """Tests a request for succesfull favoriting an article
+        :return:
+        """
+        response = self.client.post('/api/articles/', self.article,
+                                    HTTP_AUTHORIZATION='Token ' + self.token,
+                                    format='json')
+        result = json.loads(response.content)
+        slug = result["article"]["slug"]
+        self.client.post('/api/articles/' + slug + '/favorite',
+                         HTTP_AUTHORIZATION='Token ' + self.token,
+                         format='json')
+
+        response_article = self.client.delete('/api/articles/' + slug + '/favorite',
+                                              HTTP_AUTHORIZATION='Token ' + self.token,
+                                              format='json')
+        article = json.loads(response_article.content)
+
+        self.assertIn(
+            'you have successfull deleted', str(article))
```

Comment:
Nitpick fix: `you have successfully deleted`

Reply:
Thank you